src/index.js

```diff
@@ -377,7 +377,13 @@ var recycleNode = function(node) {
     : createVNode(
         node.nodeName.toLowerCase(),
         EMPTY_OBJ,
-        map.call(node.childNodes, recycleNode),
+        reduce.call(node.childNodes, function(out, node) {
+          return out.concat(
+            node.nodeType === TEXT_NODE || node.nodeType === RECYCLED_NODE
```

Comment:
It took me a while to realize you were using `RECYCLED_NODE` to mean `ELEMENT_NODE`. Haha, no problem.

Reply:
@jorgebucaran Is this going to be merged any time soon?

Reply:
@jimfranke Not now, but I will definitely revisit again later, probably after V2 is released.